cmd/activator/main.go

```diff
@@ -89,6 +89,8 @@ var (
 	masterURL  = flag.String("master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
 	kubeconfig = flag.String("kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
 
+	enableNetworkProbe = flag.Bool("enable-network-probing", false, "Whether to replace request retries with network level probing.")
```

Comment:
Should the flag var name match the flag (`enableNetworkProbing` or `enable-network-probe`)?

Reply:
Done